src/cljs/main/broadfcui/common.cljs

```diff
@@ -245,3 +245,5 @@
 ;; scopes live here instead of in auth.cljs as a quick fix to avoid circular dependencies
 (def login-scopes ["email" "profile"])
 (def storage-scopes (conj login-scopes "https://www.googleapis.com/auth/devstorage.read_only"))
+
+(def has-terra-return? (string/includes? js/window.location.search "return=terra"))
```

Comment:
Does this need to be a function? Seems like this should be constant in normal usage, but I'm imagining some power user manually editing their URL bar.

Reply:
Yeah, just in case of that and anything else, I wanted it to be evaluated dynamically.

Reply:
What I mean is, right now it won't do that.

Reply:
Ah, good catch, I think I got distracted on flipping this to a defn.